systems/summit_user_guide.rst

```diff
@@ -1733,6 +1733,69 @@ about other LSF options, see the ``bsub`` man page.
 |                    |                                        | use smt2. The default level is smt4.                                             |
 +--------------------+----------------------------------------+----------------------------------------------------------------------------------+
 
+Allocation-wide Options
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``-alloc_flags`` option to ``bsub`` is used to set allocation-wide options.
+These settings are applied to every compute node in a CSM allocation. Only one
+instance of the flag is accepted, and multiple ``alloc_flags`` values should be
+enclosed in quotes and space-separated.
```

Comment:
maybe put an example here... "and space-separated, e.g. ``-alloc_flags "smt2 gpumps"``